docs/team/shuyi.md

```diff
@@ -0,0 +1,51 @@
+# Wang Shuyi - Project Portfolio Page
+
+## Overview
+#### Project: Nav@NUS
+Nav@NUS is a Command Line Interface (CLI) application that aids users in getting around NUS, tailor made for anyone 
+unfamiliar to the campus. 
+
+Nav@NUS consists of three main features, namely Route, Dine and Fav, which helps the user to 
+plan their bus routes, find dining options, and personalize the application respectively.
+
+## Summary of Contributions
+Given below are my contributions to the project.
+
+#### New features 
+* Find dining options within a faculty: `/dine`
+    + Allows the user to see all dining options within a user-specified faculty.
+    + **Justification:** This is one of the main features of the product as it helps the user find and discover dining 
+    outlets in NUS. 
+    
+* Find specific dining outlet information: `/dineinfo`
+    + Allows the user to see the location and opening hours of a user-specified dining outlet.
+    + **Justification:** This is one of the main features of the product as it helps the user find important information 
+    regarding the dining outlet of interest, such as its location and opening hours.
+    
+* List all faculties: `/faculty`
+    + Provides the user with a list of faculties in NUS.
+
+* List routes of all buses: `/allbus`
+    + Provides the user with a list containing the routes of all shuttle buses available in NUS.
+
+* List all favorite commands: `/listfav`
+    + Allows the user to see a list of all his favorite commands with indexes and descriptions.
+
+* Clear all favorite commands: `/clearfav`
+    + Allows the user to clear his list of favorite commands.
```

Comment:
Maybe you can justify why this feature is added.

Reply:
noted.